library/cartridge_get_package_info.py

```diff
@@ -96,19 +96,25 @@ def get_deb_info(package_path):
     }
 
 
-def get_tgz_info(package_path, app_name):
+def get_tgz_info(package_path):
     tnt_version = None
 
     with tarfile.open(package_path) as tar:
         names = tar.getnames()
-        if app_name not in names:
-            raise Exception("Package should contain '%s' directory" % app_name)
+        names = list(filter(lambda path: path not in ['.', '..'], names))
 
-        tarantool_binary_path = os.path.join(app_name, 'tarantool')
+        package_name = os.path.commonprefix(names)
+        if package_name == '':
+            raise Exception(
+                "Package should contain one directory with application files. "
+                "Did you create this package using Cartridge CLI?"
```

Comment:
Maybe it's better to remove question: "Please, create package using Cartridge CLI"